<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bh_laser_widget</class>
 <widget class="QWidget" name="bh_laser_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDial" name="power_dial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="notchTarget">
    <double>3.700000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="power_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>51</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="power_spinbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string> V</string>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="frequency_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>269</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency</string>
   </property>
  </widget>
  <widget class="QDial" name="frequency_dial">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="notchTarget">
    <double>3.700000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="laser_cw_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CW</string>
   </property>
  </widget>
  <widget class="QLabel" name="laser_off_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </widget>
  <widget class="QLabel" name="laser_80MHz_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>80 MHz</string>
   </property>
  </widget>
  <widget class="QLabel" name="laser_20MHz_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>20 MHz</string>
   </property>
  </widget>
  <widget class="QLabel" name="laser_50MHz_label">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>70</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>50 MHz</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
